### PR TITLE
replace encoded slashes and backslashes with '/' for the served urls

### PR DESCRIPTION
**Pull request by makkedude**

### CODE REVIEW

File: st.js [hunk 1]
```diff
@@ -502,6 +502,13 @@ Mount.prototype._loadIndex = function (p, cb) {
       if (d.size === '-') name += '/'
       var showName = name.replace(/^(.{40}).{3,}$/, '$1..>')
       var linkName = encodeURIComponent(name)
+	  
+	  // encoded slashes are /
+	  linkName = linkName.replace(/%2f|%5c/ig, '/')
```
Comment (rvagg) [inline st.js]: keep indenting the same, have you used tabs here? should be 2-space indenting in line with the containing block

File: st.js [hunk 2]
```diff
@@ -502,6 +502,13 @@ Mount.prototype._loadIndex = function (p, cb) {
...
+
+	  // back slashes are slashes
+	  linkName = linkName.replace(/[\/\\]/g, '/')
```
Comment (rvagg) [inline st.js]: probably not necessary to include `\/` in this, just `\\` should do

**Comment by makkedude** [comment]
@rvagg fixed the stuff you noted

**Comment by rvagg** [comment]
thanks @makkedude, I actually put back the `\/` replacement because I see where you got it from now and perhaps it's catching some other form of `/` magically .. whatever

compacted your replacements, added a replacement for `.`, updated deps, fixed a test and released as

**Comment by makkedude** [comment]
that is great!

**Pull request closed without merging**